ext/opentelemetry-ext-opentracing-shim/tests/test_shim.py

```diff
@@ -406,6 +406,30 @@ def test_log_kv(self):
             # biggest possible loss of precision.
             self.assertAlmostEqual(result, now, places=6)
 
+    def test_log(self):
+        """Test the `log` method on `Span` objects."""
+        with self.shim.start_span("TestSpan") as span:
+            span.log(event="foo", payload="bar")
+            self.assertEqual(
+                span.unwrap().events[0].attributes["event"], "foo"
+            )
+            self.assertEqual(
+                span.unwrap().events[0].attributes["payload"], "bar"
+            )
+            self.assertIsNotNone(span.unwrap().events[0].timestamp)
```

Comment:
Any reason for these to be inside the context manager?

Reply:
There is not. I just copied the test_log_kv format. Will update